dsp_interview_transcripts/pipeline/README.md

```diff
@@ -1,23 +1,42 @@
 
-# Basic pipeline
+# Set up
 
-1. Make sure you have a directory `data/` with the file `qual_af_transcripts.csv` in it.
+* Create a new directory in the S3 bucket `dsp-qualfml` with a short name for your project. Record this name as `project` in the config.
 
-2. Create a directory `outputs/` in the root of the repository.
+* Upload your raw data to `s3://dsp-qualfml/<your-project>/raw/`.
```

Comment:
Maybe a few words about what filetype one should use and what columns should the raw data include?